### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 13

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#301] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-10-28 110615](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/9a93ba25-af4d-4a71-8bc0-a1a4b54f6c0a)

**Comment by IonutCicio** [comment, marked as answer]
$(G, \star)$ ha una struttura di gruppo se
1. $\forall g, g', g'' \in G, g \star (g' \star g'') = (g \star g') \star g''$ _(gode della proprietà associativa)_
2. $\forall g \in G, \exists e \in G \mid g \star e = g = e \star g$ _(ha un elemento neutro)_
3. $\forall g \in G, \exists g' \in G \mid g \star g' = e = g' \star g$ _(ogni elemento ha un inverso)_

Siano $(G, \star_G)$ e $(H, \star_H)$ due gruppi. Verificare che il prodotto cartesiano $G \times H$ ha una naturale struttura di gruppo, $(G \times H, \star)$ rispetto all'operazione

$$(g, h) \star (g', h') := (g \star_G g', h \star_H h')$$

#### Proprietà associativa

Dati $(g, h), (g', h'), (g'', h'') \in G \times H$, per associatività di $\star_G$ e $\star_H$

$$(g, h) \star ((g', h') \star (g'', h'')) =$$

$$= (g, h) \star (g' \star_G g'', h' \star_H h'') =$$

$$= (g \star_G (g' \star_G g''), h \star_H (h' \star_H h'')) =$$

$$= ((g \star_G g') \star_G g'', (h \star_H h') \star_H h'') =$$

$$= (g \star_G g', h \star_H h') \star (g'', h'') =$$

$$= ((g, h) \star (g', h')) \star (g'', h'')$$

Quindi $(G \times H, \star)$ gode della proprietà associativa.

#### Elemento neutro

Consideriamo $(e_G, e_H) \mid \forall g \in G, g \star_G e_G = g = e_G \star_G g \land \forall h \in H, h \star_H e_H = h = e_H \star h$, quindi $e_G$ e $e_H$ sono rispettivamente gli elementi neutri di $G$ e di $H$

$$
\forall (g, h) \in G \times H, (g, h) \star (e_G, e_H) = (g \star_G e_G, h \star_H e_H) = (g, h) = (e_G \star_G g, e_H \star_H h) = (e_G, e_H) \star (g, h)
$$

> Si dimostra anche che tale elemento $(e_G, e_H)$ è unico con la stessa dimostrazione che si usa normalmente per dimostrare l'unicità dell'elemento neutro

Quindi esiste l'elemento neutro $(e_G, e_H)$ in $(G \times H, \star)$.

#### Esistenza dell'inverso

Sia $(g, h) \in G \times H$ e $(g', h') \in G \times H \mid g \star_G g' = e = g' \star_G g \land h \star_H h' = e_H = h' \star_H h$, considerando $(e_G, e_H)$ l'elemento neutro in $(G \times H, \star)$ come visto precedentemente

$$(g, h) \star (g', h') = (g \star_G g', h \star_H h') = (e_G, e_H) = (g' \star_G g, h' \star_H h) = (g', h') \star (g, h)$$

Quindi esiste l'inverso $(g', h')$ di $(g, h)$ per ogni $(g, h)$ in $(G \times H, \star)$.

#### Conclusione

Quindi $(G \times H, \star)$, prodotto diretto di $G$ ed $H$ ha una naturale struttura di gruppo _c.v.d._

> Nota: si potrebbe provare a dimostrare che se $(G, \star_G)$ e $(H, \star_H)$ sono commutativi, anche $(G \times H, \star)$ è commutativo. Ricordo che un gruppo $(G, \star)$ è detto commutativo se $\forall g, g' \in G, g \star g' = g' \star g$

**Comment by Elia-Belli** [comment]
> Aggiungo la dimostrazione per due gruppi commutativi necessaria nell'esercizio successivo

**Proposizione**: Siano $G,H$ due **_gruppi commutativi_** allora il loro prodotto diretto ha naturale struttura di gruppo commutativo.
- @CuriousCI ha già dimostrato tutte le proprietà di gruppo che sono analoghe per due gruppi commutativi, manca solo la commutatività

- **Commutatività**: per definizione di prodotto diretto e commutatività dei gruppi $H,G$:
 $$(g,h)\star(g',h')=(g\star_Gg',h\star_Hh')=(g'\star_Gg,h'\star_Hh)=(g',h')\star(g,h)\ \square$$

**Comment by CiottoloMaggico** [comment]
Analogo @CuriousCI 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/83f9f039-8443-4f1b-aaf8-af11d57f1f80)
